frontend/src/components/buttons/Button.styled.ts

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+const Button = styled.button`
+    padding: 0.7rem 2rem;
+    border: 1px solid ${({ theme }) => theme.gray_500};
+    border-radius: 0.2rem;
+
+    color: ${({ theme }) => theme.white};
+    background: ${({ theme }) => theme.primary};
+
+    font-size: ${({ theme }) => theme.font_sm};
+    font-weight: ${({ theme }) => theme.font_regular};
+
+    cursor: pointer;
+    transition: all .2s;
+
+    &:hover {
+        background: ${({ theme }) => theme.gray_hover};
+    }
+
+    @media (max-width: ${({ theme }) => theme.media_md}) {
+        width: 100%;
+        padding: 1rem;
+    }
+`;
+
+const PrimaryButton = styled(Button)`
+    background: ${({ theme }) => theme.primary};
+
+    transition: all .2s;
+
+    &:hover {
+        background: ${({ theme }) => theme.primary_hover};
+    }
+
+    @media (max-width: ${({ theme }) => theme.media_md}) {
+        width: 100%;
+        padding: 1rem;
+    }
+`;
+
+const SecondaryButton = styled(Button)`
+    background: ${({ theme }) => theme.gray_200};
+    
+    transition: all .2s;
+
+    &:hover {
+        background: ${({ theme }) => theme.gray_hover};
+    }
+
+    @media (max-width: ${({ theme }) => theme.media_md}) {
+        width: 100%;
+        padding: 1rem;
+    }
+`;
+
+const TertiaryButton = styled(Button)`
```

Comment:
This is unused, why this is leave here?

Reply:
every properly designed ui has 3 types of buttons. even if 'tertiary' is not used right now it doesn't mean it won't be needed in the future. treat this as a boilerplate for future use.